README.rst

```diff
@@ -523,8 +525,6 @@ Choose from the list of available configurations:
 The ``--dry-run`` option displays the files that need to be
 fixed but without actually modifying them:
 
-.. code-block:: bash
```

Comment:
@stof I haven't installed it via PhpStorm. You did asked which editor I'm using. I've installed it via `brew`. 

Didn't know about the input option. Thanks.